src/domain/notifications/v2/notifications.repository.integration.spec.ts

```diff
@@ -325,16 +325,18 @@ describe('NotificationsRepositoryV2', () => {
               id: In(subscriptionIds),
             },
           },
-          relations: ['notification_type'],
+          relations: ['notification_type', 'notification_subscription'],
         });
 
+      expect(subscriptionNotificationTypes.length).toBeGreaterThan(0);
+
       const upsertNotificationTypes: Array<string> = [];
       upsertSubscriptionsDto.safes.map((safe) => {
         upsertNotificationTypes.push(...safe.notificationTypes);
       });
 
       for (const subscriptionNotificationType of subscriptionNotificationTypes) {
-        upsertNotificationTypes.includes(
+        expect(upsertNotificationTypes).toContain(
```

Comment:
The `toBeGreaterThan(0)` expectation is included as a fail-fast mechanism, but since the test would behave the same without it, traversing the payload to determine the length of the notification types feels unnecessary. The second expectation effectively ensures that all notification types in the payload are present in the array of objects retrieved from the database.

Reply:
Changed the test in the new commit:
https://github.com/safe-global/safe-client-gateway/pull/2182/commits/9d937f85928f5a1ebed611e495c75353ad6ac2d6